modules/openxr/extensions/openxr_fb_foveation_extension.cpp

```diff
@@ -51,6 +53,12 @@ OpenXRFBFoveationExtension::OpenXRFBFoveationExtension(const String &p_rendering
 	swapchain_create_info_foveation_fb.type = XR_TYPE_SWAPCHAIN_CREATE_INFO_FOVEATION_FB;
 	swapchain_create_info_foveation_fb.next = nullptr;
 	swapchain_create_info_foveation_fb.flags = 0;
+
+	if (rendering_driver == "opengl3") {
```

Comment:
Just a nitpick here, as we're about the have DX12 support as well, but no VRS support in DX12, we may need to add a protection in using this extension  in that scenario.